code/client/clrcore/External/Ped.cs

```diff
@@ -2,16 +2,16 @@
 using CitizenFX.Core.Native;
 using System.Security;
 
-#if MONO_V2
-using CitizenFX.Core;
+#if MONO_V2

```

Comment:
What happened here, can you undo this?

Reply:
Oh no idea what happened there... can do

Reply:
All of these are updates to line endings, VS is being a bit stupid and not letting me just stage a single line...